testsuite/integration/pom.xml

```diff
@@ -271,6 +271,15 @@
             </modules>
         </profile>
 
+        <!-- -Dts.elytron-oidc-client-->
+        <profile>
+            <id>ts.elytron-oidc-client</id>
```

Comment:
Just a minor comment, may be good to use `ts.integ.group.elytron-oidc-client` here just to be consistent with the other ids.

Reply:
Changed